From c8197b01b15509c4f8e32818d20adc19ea3cef62 Mon Sep 17 00:00:00 2001
From: pmandri <padmashree.mandri@intel.com>
Date: Mon, 6 Sep 2021 15:43:20 +0530
Subject: [PATCH] Upadte Mixer parameters for audio

Due to changes in audio_route for Android S, it expects 2 values
for each control otherwise it will throw the error "expect 2 values
but only 1 specified for ctl" and set respective ctl value to Zero
in tinymix. So updating xml parameters accordingly.

Tracked-On: OAM-99189
Signed-off-by: pmandri <padmashree.mandri@intel.com>
---
 caas/audio/default/mixer_paths_0.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/caas/audio/default/mixer_paths_0.xml b/caas/audio/default/mixer_paths_0.xml
index 0be20d7..538f79a 100644
--- a/caas/audio/default/mixer_paths_0.xml
+++ b/caas/audio/default/mixer_paths_0.xml
@@ -1,18 +1,18 @@
 <mixer>
   <!-- These are the initial mixer settings -->
-  <ctl name="Master Playback Volume" value="87" />
-  <ctl name="Master Playback Switch" value="1" />
+  <ctl name="Master Playback Volume" value="87 87" />
+  <ctl name="Master Playback Switch" value="1 1" />
   <ctl name="Headphone Playback Switch" value="0 0" />
   <ctl name="Speaker Playback Switch" value="0 0" />
   <ctl name="Capture Switch" value="0 0" />
 
   <path name="speaker">
-    <ctl name="Master Playback Switch" value="1" />
+    <ctl name="Master Playback Switch" value="1 1" />
     <ctl name="Speaker Playback Switch" value="1 1" />
   </path>
 
   <path name="headphone">
-    <ctl name="Master Playback Switch" value="1" />
+    <ctl name="Master Playback Switch" value="1 1" />
     <ctl name="Headphone Playback Switch" value="1 1" />
   </path>
 
-- 
2.17.1

